src/objects/SkinnedMesh.js

```diff
@@ -182,6 +195,83 @@ SkinnedMesh.prototype = Object.assign( Object.create( Mesh.prototype ), {
 
 		return new this.constructor( this.geometry, this.material, this.skeleton.useVertexTexture ).copy( this );
 
+	},
+
+	toJSON: function ( meta ) {
+
+		var isRootObject = ( meta === undefined || meta === '' );
+
+		if ( isRootObject ) {
+
+			meta = {
+				geometries: {},
+				materials: {},
+				textures: {},
+				images: {},
+				skeletons: {}
+			};
+
+		}
+
+		var data = THREE.Mesh.prototype.toJSON.call( this, meta );
+
+		if ( isRootObject ) {
+
+			data.metadata = {
+				version: 4.4,
+				type: 'SkinnedMesh',
+				generator: 'SkinnedMesh.toJSON'
+			};
+
+
+		}
+
+		var object = data.object;
+
+		object.bindMode = this.bindMode;
+		object.bindMatrix = this.bindMatrix.toArray();
+
+		if ( meta.skeletons[ this.skeleton.uuid ] === undefined ) {
+
+			meta.skeletons[ this.skeleton.uuid ] = this.skeleton.toJSON( meta );
+			object.skeleton = this.skeleton.uuid;
+
+		}
+
```

Comment:
@mrdoob 

I won't need `SkinnedMesh.toJSON()` if I move the above lines to `Object3D.toJSON()`
but feeling like they're too `SkinnedMesh/Skeleton` specific for `Object3D`...

What do you think?